app/views/locations/welcome.html.erb

```diff
@@ -5,5 +5,5 @@
       <p><strong><%= _("Notice") %></strong> <%= _("You must create a location before continuing.") %></p>
   </div>
   <p><%= _("Locations are used to manage and differentiate the physical place where a system managed via Foreman is housed.") %></p>
-  <p><%= _("For example, there might be a data center in London and one in Virginia; these are locations. Locations also work nicely when used in tandem with ") %> <%= link_to(_("organizations"), :controller => "organizations", :action => "index") %>.</p>
+  <p><%= (_("For example, there might be a data center in London and one in Virginia; these are locations. Locations also work nicely when used in tandem with %s.") % link_to(_("organizations"), :controller => "organizations", :action => "index")).html_safe %></p>
```

Comment:
Use a named sprintf ID here, i.e. %{organizations} to give translators additional context.  ditto in the other part of the patch.